util/src/main/scala/com/scalableminds/util/geometry/Vec3Int.scala

```diff
@@ -37,7 +37,7 @@ case class Vec3Int(x: Int, y: Int, z: Int) {
 
   override def toString: String = s"($x, $y, $z)"
 
-  def toMagLiteral(allowScalar: Boolean = false): String =
+  def toMagLiteral(allowScalar: Boolean): String =
```

Comment:
:thinking:  in some occurrences `toMagLiteral` you already changed the parameter and now made it mandatory to avoid bugs, correct?
If yes, what was the actual bug fixed here? I cannot find it in the pr description :thinking: